src/main/java/io/cdap/plugin/gcp/bigquery/util/BigQueryUtil.java

```diff
@@ -676,7 +676,7 @@ public static void validateDataset(String dataset, String datasetPropertyName, F
    */
   public static void validateTable(String table, String tablePropertyName, FailureCollector collector) {
     // Allowed character validation for table name as per https://cloud.google.com/bigquery/docs/tables
-    String errorMessage = "Table name can only contain letters (lower or uppercase), numbers, '_' and '-'.";
+    String errorMessage = "Table name can only contain letters (lower or uppercase), numbers and '_'.";
```

Comment:
Please revert this change, according to bigquery public documentation: https://cloud.google.com/bigquery/docs/tables#table_naming

BQ table names can have hyphens (`-`) why are we making this change, was it done by mistake?

Reply:
Reverted. It was done by mistake